### ISSUE TITLE
 `cast_value': undefined method `to_i' for :"420":Symbol (NoMethodError)

### ISSUE THREAD

**Issue by thnukid** [Lackoftactics/facebook_data_analyzer#24]
When I run my archive against the now `master`, I receive the following:
```
$ ruby analyze_facebook_data.rb my_archive
```

```Analyzing Messages |Time: 00:00:05 | ============================================================================================================================================================= | Time: 00:00:05
/Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/worksheet/cell.rb:466:in `cast_value': undefined method `to_i' for :"420":Symbol (NoMethodError)
Did you mean?  to_s
	from /Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/worksheet/cell.rb:112:in `value='
	from /Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/worksheet/cell.rb:49:in `initialize'
	from /Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/worksheet/row.rb:156:in `new'
	from /Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/worksheet/row.rb:156:in `block in array_to_cells'
	from /Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/worksheet/row.rb:151:in `each'
	from /Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/worksheet/row.rb:151:in `each_with_index'
	from /Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/worksheet/row.rb:151:in `array_to_cells'
	from /Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/worksheet/row.rb:35:in `initialize'
	from /Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/worksheet/worksheet.rb:401:in `new'
	from /Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/worksheet/worksheet.rb:401:in `add_row'
	from facebook_data_analyzer/classes/analyzeables/messages.rb:133:in `block (2 levels) in conversation_ranking_sheet'
	from facebook_data_analyzer/classes/analyzeables/messages.rb:130:in `each'
	from facebook_data_analyzer/classes/analyzeables/messages.rb:130:in `block in conversation_ranking_sheet'
	from /Users/user/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/axlsx-3.0.0.pre/lib/axlsx/workbook/workbook.rb:279:in `add_worksheet'
	from facebook_data_analyzer/classes/analyzeables/messages.rb:123:in `conversation_ranking_sheet'
	from facebook_data_analyzer/classes/analyzeables/messages.rb:73:in `export'
	from analyze_facebook_data.rb:46:in `block in <main>'
	from analyze_facebook_data.rb:44:in `each'
	from analyze_facebook_data.rb:44:in `<main>'
```

**Comment by Lackoftactics** [comment]
I tested it yesterday against my archive and it worked. I am wondering if that could be issue with new feature added by @northcott-j

**Comment by Lackoftactics** [comment]
@northcott-j it looks like something in your code broke it. I'm reverting that temporary as the library has to function properly without users being affected.

**Comment by northcott-j** [comment]
@Lackoftactics I think the issue isn't with my last PR, but the larger refactor to use my classes that was merged a few PR's ago. 

It looks like the issue is because I used symbols as keys instead of strings as keys for storing conversations. And the error is thrown when the xlsx gem tries to coerce a value to a cell type, if a conversation name is numbers only, it tries to make it an integer, but symbols don't have to_i.

Fix would be to either use strings as keys or declare types when making the workbook
